oss-internship-2020/sapi_lodepng/examples/main_unit_test.cc

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+#include "helpers.h"
+#include "sandbox.h"
+#include "sandboxed_api/util/status_matchers.h"
+
+using ::sapi::IsOk;
+using ::testing::Eq;
+using ::testing::IsTrue;
+using ::testing::NotNull;
+
+namespace {
+
+TEST(HelpersTest, CreateTempDirAtCWD) {
+  const std::string images_path = CreateTempDirAtCWD();
+  EXPECT_THAT(sandbox2::file_util::fileops::Exists(images_path, false),
+              IsTrue());
+
+  ASSERT_THAT(sandbox2::file_util::fileops::DeleteRecursively(images_path),
+              IsTrue());
+}
+
+TEST(HelpersTest, GenerateValues) {
+  EXPECT_THAT(GenerateValues().size(), Eq(kImgLen));
+}
+
+TEST(LodePngTest, Init) {
+  const std::string images_path = CreateTempDirAtCWD();
```

Comment:
Nit: You are checking whether `CreateTempDirAtCWD()` creates an empty folder above, but you don't check this here and you're still checking whether the `DeleteRecursively()` succeeds. Either check both or none, be consistent :)